examples/number_counts/number_counts.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
+
+import os
+
+from firecrown.likelihood.gauss_family.statistic.cluster_number_counts import (
+    ClusterNumberCounts,
+)
+from firecrown.models.ccl_density import CCLDensity
+from firecrown.likelihood.gauss_family.gaussian import ConstGaussian
+
+import sacc
+
+
+def build_likelihood(_):
+    """
+    Here we instantiate the number density (or mass function) object.
+    """
+    massfunc = CCLDensity("critical", "Bocquet16", False)
+    stats = ClusterNumberCounts(
+        "cluster_counts_true_mass", "cluster_mass_count_wl", massfunc
+    )
+    stats = [stats]
+    """
+        Here we instantiate the actual likelihood. The statistics argument carry
+        the order of the data/theory vector.
+    """
+    lk = ConstGaussian(stats)
+
+    """
+        We load the correct SACC file.
+    """
+    saccfile = os.path.expanduser(
+        os.path.expandvars(
+            "${FIRECROWN_DIR}/examples/number_counts/clusters_simulated_data.sacc"
+        )
+    )
+    sacc_data = sacc.Sacc.load_fits(saccfile)
+
+    """
+        The read likelihood method is called passing the loaded SACC file, the
+        two-point functions will receive the appropriated sections of the SACC
+        file and the sources their respective dndz.
```

Comment:
This comment should be adapted to the cluster count case.

Reply:
Yes. I fixed it